README.md

```diff
@@ -64,6 +64,23 @@ A libp2p-websocket-star address, using the signalling server we provide, looks l
 Note: The address above indicates WebSockets Secure, which can be accessed from both http and https.
 
 
+### Using WSS
+
+To be able to interact with a rendezvous server from an HTTPS site, you will need to use websocket secure. To host a secure websocket server, you must provide a keypair to the server.
+
+#### Using Key/Cert
+
+```bash
+> rendezvous --key="path/to/key.key" --cert="path/to/cert.cert"
+```
+
+#### Using Key/Cert
```

Comment:
Having the exact same heading twice seems wrong. Maybe merge them or rename the headings

Reply:
Oops. That's what I get for rushing. I appreciate the review